hokusai/kubectl.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import yaml
+
+from hokusai.common import shout
+
+class Kubectl(object):
```

Comment:
It feels a bit weird for all these classes to live in the same namespace (Kubectl, ECR, Deployment) as they don't have a great deal in common.